tools/build-release-tarball

```diff
@@ -27,13 +39,23 @@ if [ -z "${1:-}" ]; then
 fi
 
 version="$1"
+if [[ "$version" =~ ^[0-9]\.[0-9]+$ ]]; then
```

Comment:
So we care about 4.11 but not 11.4? I’d just do `^[0-9.]*$` (this condition is in negative position and should err on the loose side, not the strict side).

Could join this with the following line using `&&`.